py/picca/fitter2/chi2.py

```diff
@@ -203,6 +211,7 @@ def fastMC(self):
                 if not p in self.fast_mc:
                     self.fast_mc[p] = []
                 self.fast_mc[p].append([v, best_fit.errors[p]])
+            self.fast_mc['chi2'].append(best_fit.fval)
```

Comment:
self.fast_mc[p].append([ sp.append(v,chi2), sp.append(best_fit.errors[p],0.)])

Reply:
I don't see how your suggestion fits in here

Reply:
it doesn't anymore.

Reply:
How do you mean?

Reply:
forget about my comment